src/cascadia/TerminalApp/TerminalPage.h

```diff
@@ -94,15 +94,15 @@ namespace winrt::TerminalApp::implementation
         void _HookupKeyBindings(TerminalApp::AppKeyBindings bindings) noexcept;
         void _RegisterActionCallbacks();
 
-        void _UpdateTitle(std::shared_ptr<Tab> tab);
-        void _UpdateTabIcon(std::shared_ptr<Tab> tab);
+        void _UpdateTitle(winrt::TerminalApp::Tab tab);
```

Comment:
since we're in the TerminalApp namespace we may even be able to just call this `Tab`

Reply:
unless you explicitly want to capture a reference to the tab, you can still use `const Tab&`